cmd/config.sample.toml

```diff
@@ -1,18 +1,28 @@
-# Redis addresses are denoted as: redis://password@host:port/db_num
-
-[job_queue]
-broker_address = "redis://127.0.0.1:6379/1"
-
-# This is where Machinery stores it's job states.
-# In Machinery's terminology, this is its "result backend"
+[job_queue.broker]
+address = ["localhost:6379"]
```

Comment:
`addrs` or `addresses` (plural) would be make is clearer.

Reply:
Yepp, using `addresses` since we're not using shorthands elsewhere (`password` / `pass`, etc)